docs/development.rst

```diff
@@ -571,12 +571,48 @@ You can use ``docker`` to locally build an image, but it requires root access:
 Documentation
 *************
 
-Documentation is written using `Sphinx <http://www.sphinx-doc.org/en/stable/>`_
-and contained in the ``docs`` directory. It is written in the
-`reStructuredText <http://docutils.sourceforge.net/rst.html>`_ format and
-is deployed automatically to `readthedocs <https://readthedocs.org/>`_. To
-build the documentation locally run ``make`` in the ``docs`` directory.
-This should build the HTML documentation in ``docs/_build/html/``.
+To get started writing documentation for msprime, first follow the `Quickstart <https://tskit-dev.github.io/msprime-docs/main/development.html#quickstart>`_. Once you have created and checked out a "topic branch", you are ready to start editing the documentation.
```

Comment:
Better to use sphinx cross references here, it'll be something like ":ref:`sec_development_quickstart`", but you might need to check the actual label is defined. This is more robust, especially as we're moving the docs into ``tskit.dev`` sometime soon.